### PR TITLE
Add jqueryui

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by dionyziz** [comment]
Please include library upgrade knowledge.

File: views/header.php [hunk 1]
```diff
@@ -35,6 +35,9 @@
             includeScript( 'user/create' );
             includeScript( 'session/create' );
         ?>
+        <link rel="stylesheet" type="text/css" href="static/jquery-ui/jquery-ui.min.css" />
+        <link rel="stylesheet" type="text/css" href="static/jquery-ui/jquery.ui.theme.css" />
```
Comment (dionyziz) [inline views/header.php]: `includeStyle` with `../jquery-ui`?

File: views/header.php [hunk 2]
```diff
@@ -35,6 +35,9 @@
...
+        <script type="text/javascript" src="static/jquery-ui/jquery-ui-1.10.4.min.js"></script>
```
Comment (dionyziz) [inline views/header.php]: `includeScript` with `../jquery-ui`?

File: mockups/game/game.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```
Comment (dionyziz) [inline mockups/game/game.html:1]: Is this file new? Where was this previously?

Comment (vitsalis) [inline mockups/game/game.html:1]: you didn't add it on github.

Comment (dionyziz) [inline mockups/game/game.html:1]: I see, thanks :)

**Pull request merged**